Certificates/cert-gen.py

```diff
@@ -0,0 +1,106 @@
+# Requires inkscape in path and PDF module
+import argparse
+import csv
+import numpy as np
+import os
+import pathlib
+import sys
+from PDF import PdfFileReader, PdfFileWriter
+from subprocess import Popen
+
+def argument_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        'template_svg',
+        help="Input svg template with text matching 'NAME OF STUDENT IN CAPS' and 'NAME OF SCHOOL IN CAPS' for "
+             "replacement with the students' names and schools'/teams' names, respectively, "
+             "on the produced certificates.",
+        type=pathlib.Path,
+    )
+    parser.add_argument(
+        'input_data',
```

Comment:
if you change `'input_data'` to `'--input_data`, you can make it optional (and therefore don't need the `default=-1`)

It means that `options.input_data` is `None` if unset (so you'll need to change the `== -1` later on)

Reply:
That makes sense :) if I do that for both optional arguments, then it will be possible to set an output directory without input data